libs/shared/ui/src/lib/components/badge/badge.tsx

```diff
@@ -0,0 +1,123 @@
+import { type VariantProps, cva } from 'class-variance-authority'
+import { type ComponentPropsWithoutRef, type ElementRef, forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+
+const badgeVariants = cva(['inline-flex', 'items-baseline', 'shrink-0', 'font-medium', 'rounded'], {
+  variants: {
+    color: {
+      neutral: [],
+      red: ['text-red-500'],
+      purple: ['text-purple-500'],
+      sky: ['text-sky-500'],
+      green: ['text-green-500'],
+      yellow: ['text-yellow-500'],
+      brand: ['text-brand-500'],
```

Comment:
I think you can remove the `red`, `green`, `purple`, and `yellow`. It's not planned to use it on the Console (can be added later if needed)

Reply:
They are already used in the current console for example in the pricing page, so I cannot remove them.